test/commands/addons/attach.js

```diff
@@ -83,4 +85,18 @@ Setting POSTGRES_HELLO config vars and restarting myapp... done, v10
 `))
       .then(() => api.done())
   })
+
+  it('errors if the credential flag is specified but that credential does not exist for that addon', function () {
+    nock('https://api.heroku.com:443')
+      .get('/addons/postgres-123')
+      .reply(200, {name: 'postgres-123'})
+      .get('/addons/postgres-123/config/credential:hello')
+      .reply(200, [])
+
+    return expect(cmd.run({
+      app: 'myapp',
+      args: {addon_name: 'postgres-123'},
+      flags: {credential: 'hello'}
+    }), 'to be rejected with error satisfying', new Error('Could not find credential hello for database postgres-123'))
```

Comment:
More for my education here, what does this `'to be rejected with error satisfying'` do in context of the test? It looks like just a descriptive text.